examples/deform_geometry/runScript.py

```diff
@@ -0,0 +1,163 @@
+    # Load the desired pyGeo object
+    # =========================================================================
+    # rst IGES
+    # ------------------------------ IGES File ------------------------------ #
+    geo = pyGeo(fileName="./geo/wing.igs", initType="iges")
+    geo.doConnectivity()
+    # rst IGES (end)
+
+    # =========================================================================
+    # Setup DVGeometry object
+    # =========================================================================
+    DVGeo = DVGeometry("./ffd/ffd.xyz")
+
+    # Create reference axis
+    nRefAxPts = DVGeo.addRefAxis("wing", xFraction=0.25, alignIndex="k")
+
+    # set the twist variable
+    def twist(val, geo):
+        for i in range(nRefAxPts):
+            geo.rot_z["wing"].coef[i] = val[i]
+
+    # set initial twist, we set the root twist to 5 deg so wingSurfMeshNew should reflect this
+    twist0 = [0] * nRefAxPts
+    twist0[0] = 5
+
+    # add the twist design variable to DVGeo
+    DVGeo.addGeoDVGlobal(dvName="twist", value=twist0, func=twist, lower=-10, upper=10, scale=1.0)
+
+    # =========================================================================
+    # Update pyGeo Object and output result
+    # =========================================================================
+    DVGeo.updatePyGeo(geo, "tecplot", "wingNew", nRefU=10, nRefV=10)
+
+def deform_plot3d():
+    # =========================================================================
+    # Load the desired pyGeo object
+    # =========================================================================
+    # rst plot3d
+    # ----------------------------- Plot3D File ----------------------------- #
+    geo = pyGeo(fileName="./geo/wing.xyz", initType="plot3d")
+    geo.doConnectivity()
+    geo.fitGlobal()
+    # rst plot3d (end)
+    # =========================================================================
+    # Setup DVGeometry object
+    # =========================================================================
+    DVGeo = DVGeometry("./ffd/ffd.xyz")
+
+    # Create reference axis
+    nRefAxPts = DVGeo.addRefAxis("wing", xFraction=0.25, alignIndex="k")
+
+    # set the twist variable
+    def twist(val, geo):
+        for i in range(nRefAxPts):
+            geo.rot_z["wing"].coef[i] = val[i]
+
+    # set initial twist, we set the root twist to 5 deg so wingSurfMeshNew should reflect this
+    twist0 = [0] * nRefAxPts
+    twist0[0] = 5
+
+    # add the twist design variable to DVGeo
+    DVGeo.addGeoDVGlobal(dvName="twist", value=twist0, func=twist, lower=-10, upper=10, scale=1.0)
+
+    # =========================================================================
+    # Update pyGeo Object and output result
+    # =========================================================================
+    DVGeo.updatePyGeo(geo, "tecplot", "wingNew", nRefU=10, nRefV=10)
+
+if __name__ == "__main__":
+    if len(sys.argv) != 1:
```

Comment:
We generally prefer `argparse` over raw `sys.argv` parsing.

Reply:
Fixed.

Reply:
This can actually be improved:
```
parser.add_argument("--output_type", type=str, default="iges", choices=["iges", "plot3d", "liftingsurface"])
```
Then you don't need the `else` block.

Reply:
Thanks for the recommendation -- updated.

Reply:
What I meant was that because you specified `choices`, if the value given by the user does not fall into those, then `argparser` itself will [raise the error](https://docs.python.org/3/library/argparse.html#choices). So there is no need for the `else` block altogether since any error would be raised at the `parse_args()` call already.

Reply:
Yes, sorry I forgot to delete that line after updating the parsing line. I fix that.